src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php

```diff
@@ -244,16 +268,16 @@ private function sendTokenEmail(UserInterface $user, $from, $to)
             throw new TokenEmailsLimitReachedException(self::MAX_NUMBER_EMAILS, $user);
         }
         $mailer = $this->get('mailer');
-        $translator = $this->get('translator');
         $em = $this->getDoctrine()->getManager();
         $message = $mailer->createMessage()
             ->setSubject(
-                $translator->trans(static::$emailSubjectKey, [], static::$translationDomain)
+                $this->getSubject()
             )
             ->setFrom($from)
             ->setTo($to)
             ->setBody(
-                $translator->trans(static::$emailMessageKey, [], static::$translationDomain) . PHP_EOL .
+                $this->getMessage()
+                . PHP_EOL .
                 $this->generateUrl(static::$resetRouteId, ['token' => $user->getPasswordResetToken()], true)
```

Comment:
Shouldn't `generateUrl` be part of the `getMessage` method? Could easily imagine that some people want to embed it in the message, and not just append it to the end.